### PR TITLE
Register device fcm

### PR DESCRIPTION
**Pull request by 04HuynhM**
Endpoints for registering a client device (unit) and updating a unit's fcm token

### CODE REVIEW

File: functions/units/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const dbref = admin.firebase().ref();
```
Comment (thefnox) [inline functions/units/index.js]: Something tells me this should probably be inside the functions

File: functions/units/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        })
+    }
+});
+
+exports.updateFcmToken = functions.https.onRequest((req, res) => {
+    if(req.method !== "POST") {
+        return res.status(400).json({
+            message: "Bad request, POST Required"
+        })
+    } else {
```
Comment (thefnox) [inline functions/units/index.js]: You're exiting the function here, no need for else.

File: functions/units/index.js (hunk 1 above)
Comment (04HuynhM) [reply functions/units/index.js]: You mean the firebase database reference? What is your reasoning behind that?

File: functions/units/index.js (hunk 2 above)
Comment (04HuynhM) [reply functions/units/index.js]: Got it

**Review by thefnox — APPROVED** [review]

**Pull request merged**